vlo-importer/src/main/java/eu/clarin/cmdi/vlo/importer/CMDIData.java

```diff
@@ -9,12 +9,17 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import eu.clarin.cmdi.vlo.FacetConstants.KEY;
 import eu.clarin.cmdi.vlo.FacetConstants;
+import eu.clarin.cmdi.vlo.config.FieldNameService;
 
 /**
  * Represents a document of CMDI data.
  */
 public class CMDIData {
+    private FieldNameService fieldNameService;
```

Comment:
I'd make this `final`, which IMO makes the purpose/behaviour of this field easier to understand when first reading the code; same thing in a couple of other places where `fieldNameService` is injected via the constructor and never modified (`FacetMappingFactory`, `LicenseTypePostNormalizer`, `SolrDocumentQueryFactoryImpl` to name a few) .